data/inputs/gri30_ion.cti

```diff
@@ -0,0 +1,2192 @@
+#
```

Comment:
- Rather than introducing a full new copy of the GRI mechanism with a few additions, you can import the data for the existing species and reactions from `gri30.cti` and then just add the new reactions and species.
- The source for the new reactions and species data should be given